examples/destination/java/src/main/java/destination/DestinationServiceImpl.java

```diff
@@ -112,4 +115,9 @@ public void writeBatch(WriteBatchRequest request, StreamObserver<WriteBatchRespo
         responseObserver.onNext(WriteBatchResponse.newBuilder().setSuccess(true).build());
         responseObserver.onCompleted();
     }
+
+    private static String getChangesFromRequest(AlterTableRequest request) {
+        final List<SchemaDiff> changes = request.getChangesList();
+        return changes.stream().map(AbstractMessage::toString).collect(Collectors.joining(", "));
```

Comment:
No need to create a separate method for this

Reply:
updated